### PR TITLE
Upgrade radios to NanoUI

### PR DESCRIPTION
**Pull request by tigercat2000**
Things changed:
 - All radios now use nanoUI
 - Syndicate mode for any radios with syndicate access!
 - Electropacks also upgraded, since they are technically radios
 - Fancy as a motherfucker

Glorious screenshots:
trwt
trwt
trwt
trwt
trwt
trwt
trwt
trwt

### CODE REVIEW

File: code/modules/nano/interaction/default.dm [hunk 1]
```diff
@@ -43,6 +43,8 @@
 		var/obj/machinery/Machine = src_object
 		if(Machine.emagged)
 			return STATUS_INTERACTIVE
+	if(istype(src_object, /obj/item/device/radio/borg/syndicate)) //Not ideal, but they can't access their own radio otherwise.
```
Comment (tigercat2000) [inline code/modules/nano/interaction/default.dm]: If anyone else has a good idea how else to do this, please say

Comment (PsiOmegaDelta) [inline code/modules/nano/interaction/default.dm]: On Bay Syndicate borgs no longer has special NanoUI treatment. Fairly big changes for it to work tho, see https://github.com/Baystation12/Baystation12/pull/10694.

File: code/game/objects/items/devices/radio/radio.dm [hunk 2]
```diff
@@ -703,32 +710,40 @@ var/global/list/default_medbay_channels = list(
...
+	if(syndie)
+		data["useSyndMode"] = 1
+
+	data["has_loudspeaker"] = 1
+	data["loudspeaker"] = !shut_up
+	data["has_subspace"] = 1
+	data["subspace"] = subspace_transmission
+
+	ui = nanomanager.try_update_ui(user, src, ui_key, ui, data, force_open)
+	if(!ui)
```
Comment (Markolie) [inline code/game/objects/items/devices/radio/radio.dm]: I'd recommend making them a little less wide.

File: code/modules/nano/interaction/default.dm (hunk 1 above)
Comment (Markolie) [inline code/modules/nano/interaction/default.dm]: Add CanUseTopic() to the /syndicate radio subchild in the radio.dm file, and then add the check for Syndicate borgs there and have it return 1.

**Comment by Spacemanspark** [comment]
Yay.

**Comment by tigercat2000** [comment]
New width: bluie

File: code/game/objects/items/devices/radio/radio.dm (hunk 2 above)
Comment (tigercat2000) [inline code/game/objects/items/devices/radio/radio.dm]: Now 400 by 550 instead of 550 by 500

**Pull request merged**